appinfo/routes.php

```diff
@@ -13,8 +13,6 @@
 	   ['name' => 'main#home', 'url' => '/home', 'verb' => 'GET'],
 	   ['name' => 'main#keywords', 'url' => '/keywords', 'verb' => 'GET'],
 	   ['name' => 'main#categories', 'url' => '/categories', 'verb' => 'GET'],
-	   ['name' => 'main#category', 'url' => '/category/{category}', 'verb' => 'GET'],
```

Comment:
Is this really no more needed? I just found [this line](https://github.com/nextcloud/cookbook/blob/master/src/router/index.js#L28) but I am not sure if this is related (not JS/VUE expert). Just the red warn lamp when changing the API.

Reply:
The API is, at the moment, a mix of the old PHP routes (which are reachable only through manually writing them on the address line) and the new JSON API routes. I have not just gone ahead and pruned it because I am not familiar with the back-end and don't want to remove anything that is actually still in use. Unless the goal is to maintain both the Vue front-end and a backup synchronous front-end for people who cannot or refuse to use javascript, the obsolete routes should be removed at some point. Maintaining two front-ends is too big an undertaking for me at least, so unless there is someone else on the team to maintain the PHP template side, I suggest we get rid of it sooner rather than later. Again, this should probably have its own issue.

And yes, the line you spotted in the Vue router does replace the line that was removed.

Reply:
In fact, you moved the routes to the `/api` prefix [here](https://github.com/nextcloud/cookbook/pull/318/files#diff-0e033fd8797499a87a6bf2ffccef5c3cR28-R29). This is perfectly fine for me. I was just curious why in the vue router the prefix is not mentioned.